### ISSUE TITLE
Errors when you don't have Android tools

### ISSUE THREAD

**Issue by upgundecha** [sudharsan-selvaraj/appium-dashboard-plugin#46]
Looks like there is a hard dependency for Android tools even if I want to run only on iOS setup

```
[Appium] Attempting to load plugin appium-dashboard...
[appium-dashboard] Unable to create adb instance
[appium-dashboard] Error: Neither ANDROID_HOME nor ANDROID_SDK_ROOT environment variable was exported. Read https://developer.android.com/studio/command-line/variables for more details
```

```
An unknown server-side error occurred while processing the command. Original error: Service with "adb" identifier was not found in the container. Register it before usage via explicitly calling the "Container.set" function or using the "@Service()" decorator. (Selenium::WebDriver::Error::UnknownError)
      UnknownError: An unknown server-side error occurred while processing the command. Original error: Service with "adb" identifier was not found in the container. Register it before usage via explicitly calling the "Container.set" function or using the "@Service()" decorator.
          at getResponseForW3CError (/Users/unmeshgundecha/.nvm/versions/node/v17.0.1/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/errors.js:804:9)
          at asyncHandler (/Users/unmeshgundecha/.nvm/versions/node/v17.0.1/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:370:37)
      ./features/support/env.rb:49:in `Before'
```

Android dependency can be made soft and support iOS runs without needing to install Android tools.

**Comment by sudharsan-selvaraj** [comment]
Looks like a valid issue. Thanks for reporting @upgundecha 👍

**Comment by sudharsan-selvaraj** [comment]
Fixed it as part of #50

**Comment by sudharsan-selvaraj** [comment]
@iepoch , the fix is not yet released to npm. I will make a new release tomorrow and you won't see the issue.
